### PR TITLE
Fixed the laravel sample application

### PR DESCRIPTION
**Pull request by imonroe**
What I changed:
- in the Dockerfile, it called for `mysql-client`, but the new php-7.2 image needs that to be changed to `mariadb-client`.  See: https://stackoverflow.com/questions/57048428/e-package-mysql-client-has-no-installation-candidate-in-php-fpm-image-build-u
- in the `./configuration/nginx/conf.d/app.conf` file, there is a note that the `fastcgi_pass` variable needs to be changed to reflect the name of the container.  I changed it to `project_laravel` to ensure it would come up correctly.
- in the `docker-compose.yml` file, the context to build the app image was listed as `./laravel-app`, but that's not where the dockerfile lives.  Changed it to just `.`, which fixes the build.
I then re-tarred the file and replaced the one in the repo.

### CODE REVIEW

**Comment by githubsaturn** [comment]
Thanks @imonroe !

I have to park this PR for a few days until I find a new home for sample apps. The current way of bundling them with the main repo has made the repo way too bloated. I'll split them out and will let you know the new format.

**Comment by imonroe** [comment]
No problem.  I was tempted to exclude the `vendor` directory to reduce the size of the tar, but I figured if it was there, it was probably for a good reason.

**Comment by sleeyax** [comment]
Sooooooo is this PR ever gonna get merged?

**Comment by githubsaturn** [comment]
TLDR:
No. I just kept the PR open for people who are interested. You can download the tar from the source of the PR: https://github.com/imonroe/caprover/blob/fix-laravel/captain-sample-apps/laravel.tar

The tar file is too large to be merged into the main repo. I was planning on creating a separate repo for sample apps, but I haven't gotten to it yet. Alternatively, some people started hosting their own sample repos and added them to the [Community Apps Section](https://caprover.com/docs/sample-apps.html#community-apps). Feel free to do that.

**Comment by imonroe** [comment]
I likely should update my fork to update PHP to 7.3 and Laravel to 7.x  I'll see if I can do that this weekend.

**Comment by githubsaturn** [comment]
closing old PRs
